packages/formation-react/src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx

```diff
@@ -32,7 +32,7 @@ describe('<AdditionalInfo/>', () => {
     wrapper.unmount();
   });
   it('renders both children when open is true', () => {
-    const first = wrapper.find('ExpandingGroup').props();
+    const first = wrapper.find(ExpandingGroup).props();
```

Comment:
I don't know why this started failing on my branch- suspect there might be a bigger issue with our unit tests.